app/konnect/compatibility.md

```diff
@@ -23,6 +23,15 @@ title: Compatibility
 | {{site.ee_product_name}} 2.4.x or earlier | <i class="fa fa-times"></i>    | --
 
 
+## {{site.mesh_product_name}} compatibility
+
+To use [Mesh Manager](/konnect/mesh-manager/), you must also use a compatible version of {{site.mesh_product_name}}. 
+
+|                                  | {{site.konnect_saas}} | First supported patch version
+|--------------------------------|:---------------------:|-----------------------------
+| {{site.mesh_product_name}} 2.3.x | <i class="fa fa-check"></i> | 2.3.0
```

Comment:
Let's start straight with 2.4.x 
```suggestion
| {{site.mesh_product_name}} 2.4.x | <i class="fa fa-check"></i> | 2.4.1
```

Just so that folks use the latest